src/Content.php

```diff
@@ -678,12 +678,18 @@ public function getDecodedValue($name)
                     // Parse the field as Markdown, return HTML
                     $value = \ParsedownExtra::instance()->text($value);
 
+                    $config = $this->app['config']->get('general/htmlcleaner');
+                    $allowed_tags = !empty($config['allowed_tags']) ? $config['allowed_tags'] :
```

Comment:
Might be safer to set the defaults for those in [`Config::getDefaults()`](https://github.com/bolt/bolt/blob/master/src/Config.php#L7350) in case someone updates one set, but forgets the other